### ISSUE TITLE
Afterimage wonkyness

### ISSUE THREAD

**Issue by LasombraDemon** [ikemen-engine/Ikemen-GO#984]
### Describe the bug

This code causes a blinking afterimage in MUGEN, and a solid one in IKEMEN Go, reported by Kusaalst.

### To Reproduce

[State 0 Shadow]
type = AfterImage
trigger1 = !time
time = 2
length = 8
palcolor = 0
palbright = 0,0,110
palpostbright = 10,10,97
palcontrast = 220,220,220
paladd = 0,0,0
palmul = 1,1,1
framegap = 2

[State 0 Shadow]
type = Afterimagetime
trigger1 = time%2 = 0 && time >= 0
time = 2

### Expected behavior

Afterimage should blink as in first video, is instead solid.

### Screenshots / Video

MUGEN: https://streamable.com/fdfmlr

IKEMEN Go 0.98.2: https://streamable.com/gvq7yt

### Engine Version (or source code date)

0.98.2

### Operating system

Windows

**Comment by K4thos** [comment]
please upload the character and provide the inputs to do the move.

**Comment by LasombraDemon** [comment]
Character used in both examples.

[Iori_KusaaIst.zip](https://github.com/ikemen-engine/Ikemen-GO/files/10714933/Iori_KusaaIst.zip)

**Comment by potsmugen** [comment]
That looks like it's exploiting some bug in the Mugen engine. There's no parameter to make them blink. So it's like you're bringing the afterimage back up in the same frame that it goes away to give that impression.

**Comment by LasombraDemon** [comment]
The author of the char is Kusaalst, he's been digging deep into the engine and pointed to this one.

**Comment by potsmugen** [comment]
Yeah in that case it looks like the same Mugen bug (feature?) is hiding a mistake in the code whereas Ikemen behaves as expected.
